ft(getmael):get all meals

olorunwalawrence opened the pull request:
#### What does this PR do?
Get all meals
#### Description of the task to be completed
- create meal controllers
- create routes
- helper function to get all meals

#### How to manually test this PR
- clone the [repo ](https://github.com/olorunwalawrence/book-a-meal)
- on your terminal, run `git checkout  get-all-meal-163848832`
- on your terminal, run npm install to install dependencies
- on your terminal, run npm run start:dev to start the application.

#### What is the story identity?
#163848832

olorunwalawrence committed:
ft(getmael):get all meals
- create meal controllers
- helper function to get all meals
[Delivers #163848832]

Billmike commented on app.js:
```diff
@@ -1,13 +1,14 @@
 import express from 'express';
 import bodyParser from 'body-parser';
-
+import route from './dummy/route/index';
```
Since the file name is `index`, you can leave out the index at the end of this file import. So it becomes `import route from './dummy/route`

Billmike commented on dummy/route/index.js:
```diff
@@ -0,0 +1,11 @@
+router.get('/meals', getAllMeal);
+
+// eslint-disable-next-line eol-last
+export default router;
```
Add a new line here

olorunwalawrence committed:
ft()implement mentor feed backs
- leave out index at the end of file import
- give a line space
[Delivers #163848832]
